packages/office-ui-fabric-react/src/components/TextField/MaskedTextField/MaskedTextField.tsx

```diff
@@ -0,0 +1,325 @@
+import * as React from 'react';
```

Comment:
This is missing tests. Needs a bunch of enzyme tests. And probably a snapshot test or two.

Reply:
Done.